polaris-react/src/components/ResourceList/ResourceList.scss

```diff
@@ -84,14 +84,15 @@ $item-wrapper-loading-height: 64px;
   background-color: var(--p-color-bg);
   border-radius: var(--p-border-radius-2);
   #{$se23} & {
-    min-height: 48px;
-    align-items: center;
     // This container conditionally includes a Select component for sorting
-    // We need dynamic padding values here to ensure the Select component and its container
-    // have adequate top and bottom spacing at each breakpoint.
-    padding: var(--p-space-1_5-experimental) var(--p-space-3);
+    // It's 32px above md-up and 36px below
+    // This hardcoded min-height is to ensure the Select has uniform padding when it's 36px high
+    min-height: 52px;
+    align-items: center;
+    padding: var(--p-space-2) var(--p-space-3);
```

Comment:
```suggestion
    align-items: center;
    padding: var(--p-space-2) var(--p-space-3);
    min-height: calc(
      // Always leave space for a Select component
      var(--p-control-height) +
      // And account for the top/bottom padding set above
      calc(2 * var(--p-space-2))
    );
```

Reply:
☝️ Not much point doing this as we don't have a `--p-control-height` var yet, so the hard coded value works for now.

See also: https://github.com/Shopify/polaris-summer-editions/issues/299